### ISSUE TITLE
Wrong display just after downloading and installing a map

### ISSUE THREAD

**Issue by lains** [navit-gps/navit#948]
Fresh install of navit, compiled from sources at https://github.com/navit-gps/navit/commit/541bff3b8bc13fd8ae78cfb697da3923e9e117dc
No /sdcard/navit folder.
In the internal GUI, select Tools->Map Download
Select a map covering the current position.
Once the download is finished, the display directly switches to the map view, but when trying to move around in the map, the GUI buttons reappear, showing that the GUI was still in menu mode.

![Screenshot_20191123-064728](https://user-images.githubusercontent.com/383502/69474173-fcd4b580-0dbd-11ea-8d47-1cf2fdf1f737.png)

Pressing the back button a few times returns to the map view properly.

This is maybe happening after commit https://github.com/navit-gps/navit/commit/aefa85db918cd8026814ad12802fc319dea18699 has been introduced.

**Comment by jkoan** [comment]
i have been able to reproduce this behavior.

**Comment by lains** [comment]
Please let me try again using a recent build for Android.

**Comment by lains** [comment]
Hell @jkoan, I could reproduce the issue on a build based on the last commit https://github.com/navit-gps/navit/commit/7fecfe844eed72d290b6b6301b5ae96893d52fae.
On my Android device, with an SD card inserted, I download a new map from the server.
Once finished, the menu quickly displays, then the map is displayed instead but if I swipe my finger over the map to move it, it actually draws the menu icons as my finger moves over them (see screenshot above).
Here are the logs I could capture at the end of the map download:
```
06-05 09:10:24.580  4229  4368 D NavitMapDownLoader: success
06-05 09:10:24.610  4229  4229 V NavitGraphics: onWindowFocusChanged = true
06-05 09:10:24.611   587   602 W InputMethodManagerService: Window already focused, ignoring focus gain of: com.android.internal.view.IInputMethodClient$Stub$Proxy@7216b0c attribute=null, token = android.os.BinderProxy@1d5422a
06-05 09:10:24.611  4229  4229 E NavitDialogs: onDismiss: mMapdownloaderDialog
06-05 09:10:24.611  4229  4229 D NavitMapDownLoader: mStopMe -> true
06-05 09:10:24.621  4229  4229 W navit   : warning:map_binfile:struct zip64_eoc *binfile_read_eoc64(struct file *):map file /storage/emulated/0/Android/data/org.navitproject.navit/files/Europe.bin: eocl wrong
06-05 09:10:24.977  4229  4229 V NavitGraphics: overlay_disable: 0, Parent: true
06-05 09:10:26.109  4229  4244 I art     : Background sticky concurrent mark sweep GC freed 52395(2MB) AllocSpace objects, 6(268KB) LOS objects, 34% free, 7MB/11MB, paused 824us total 740.759ms
06-05 09:10:26.124   587   701 D Sensors : LightSensor readEvents x = 16.000000, raw = 16
06-05 09:10:26.978  4229  4229 V Navit   : Handle Event
06-05 09:10:26.978  4229  4229 V Navit   : Handle Event
06-05 09:10:26.978  4229  4229 V Navit   : Handle Event
```

**Comment by lains** [comment]
And when getting these buttons displayed on the map, if I hit the "back" button, the main menu is actually displayed entirely (the map disappears).
So it really seems the map is displayed on the screen by mistake, but navit is actually still presenting the internal menu ("Tools") that was the starting point before the NavitMapDownloader activity was fired.

**Comment by lains** [comment]
Mix of internal GUI and map still exists in recent versions of Navit (tried latest git commit 45f4e0263f87591c02b571f768366cc87c14bef9).

The issue can be reproduced on Android by:
- displaying the map
- tap anywhere to display main internal GUI menu
- tap on action -> coordinates
- while coordinates screen is displayed, switch to another app and display it
- now switch back to navit
- the map is displayed again, hiding the internal GUI menu
- if you touch the screen and while keeping pressing, move your finger at the top of the display, widgets from the coordinates menu still display on top of the map
- press the back button sorts out the display, the map disappears and the action menu is displayed correctly
